components/apimgt/org.wso2.carbon.apimgt.impl/src/test/java/org/wso2/carbon/apimgt/impl/definitions/OAS2ParserTest.java

```diff
@@ -147,4 +151,42 @@ public void testRemoveResponsesObjectFromOpenAPI20Spec() throws Exception {
         Assert.assertEquals(oas2Parser.removeResponsesObject(swagger,swaggerWithoutResponsesObject),
                 oas2Parser.removeResponsesObject(swagger,swaggerWithResponsesObject));
     }
+
+    @Test
```

Comment:
If possible let's add a description.

Reply:
Since we use the same method signature as in the acutal method I guess description is not required. (Same approach followed throughout the other tests as well)